docs/UnInstall_IML.md

```diff
@@ -0,0 +1,184 @@
+[**Table of Contents**](index.md)
+
+# Uninstalling IML
+
+![clustre](md_Graphics/uninstall_sm.jpg)
+
+# **Step 1**: Uninstall the IML Manager
+
+### Stop the manager and httpd 
+
+```
+chroma-config stop
+```
+
+### Remove the Manager
+```
+chkconfig --del chroma-supervisor
+yum clean all --enablerepo=*
+yum remove chroma* fence-agents*
+rm -rf /var/lib/chroma/*
+rm -rf /usr/share/chroma-manager/
+```
+
+### Optionally, remove HTTPD
+```
+yum remove httpd
+For IEEL 3.0, 
+yum remove nginx
```

Comment:
Have you confirmed that removing the chroma-manager repo doesn't uninstall nginx?

Reply:
It shouldn't.